src/main/scripts/configure-ihs-on-dmgr.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+
+#      Copyright (c) Microsoft Corporation.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+# 
+#           http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Check required parameters
+if [ "$7" == "" ]; then 
+  echo "Usage:"
+  echo "  ./configure-ihs-on-dmgr.sh [profile] [wasUser] [wasPassword] [storageAccountName] [storageAccountKey] [fileShareName] [mountpointPath]"
+  exit 1
+fi
+profile=$1
+wasUser=$2
+wasPassword=$3
+storageAccountName=$4
+storageAccountKey=$5
+fileShareName=$6
+mountpointPath=$7
+
+source /datadrive/virtualimage.properties
+
+# Mount Azure File Share system
+mkdir -p $mountpointPath
+mkdir /etc/smbcredentials
+echo "username=$storageAccountName" > /etc/smbcredentials/${storageAccountName}.cred
+echo "password=$storageAccountKey" >> /etc/smbcredentials/${storageAccountName}.cred
+chmod 600 /etc/smbcredentials/${storageAccountName}.cred
+echo "//${storageAccountName}.file.core.windows.net/${fileShareName} $mountpointPath cifs nofail,vers=2.1,credentials=/etc/smbcredentials/${storageAccountName}.cred,dir_mode=0777,file_mode=0777,serverino" >> /etc/fstab
+
+yum install cifs-utils -y
```

Comment:
Can we do this on the image itself?

Reply:
Yes, we can. The reason why it's moved to the image is that IHS configuration is an optional step. If user doesn't configure an IHS using the offer but manually set it up for some reason, he/she can use other approaches to transfer configuration script between IHS and DMgr VMs.

Reply:
@git4rk Let's approach in this way:
1. Keep `yum install cifs-utils -y` now
1. Update cicd pipelines for both twas-nd and ihs base images to install cifs-utils preparing VM
1. Remove `yum install cifs-utils -y` from deployment scripts in cluster repo

I opened xxx to track the